lib/resources/dstu2/soarian_document_reference.yaml

```diff
@@ -0,0 +1,206 @@
+---
```

Comment:
There are a few terminologies returned in the example Doc ref that are not bound here (and @MaxPhilips , the terminology table pulls in the loinc binding from here). SecurityLabel is not a required binding, so it needs to be in the table. Content format as well. Essentially: any codified element that doesn't have a required binding should be covered in the terminology table (assuming the element is returned)

Max is correct - any field that has no binding can be removed since it won't apply.